.goreleaser.yaml

```diff
@@ -0,0 +1,59 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+    # you may remove this if you don't need go generate
+    - go generate ./...
+builds:
+  - id: daja
+    binary: daja
+    main: .
+    goos:
+      - linux
+      - darwin
+      - windows
```

Comment:
```suggestion
```
I dont know if windows will be a thing, so by now we cannot include on the build. This could help saving computing time, space and also not mislead the use of daje.